test/java/seedu/addressbook/storage/InvalidData.txt

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<AddressBook>
+    <persons>
+        <name>John Doe</name>
+
+        <email isPrivate="false">johnd@gmail.com</email>
+        <address isPrivate="false">John street, block 123, #01-01</address>
+    </persons>
+</AddressBook>
```

Comment:
In my thought, here i only check whether the xml format is valid. i.e. have relevant <Name><phone>... pairs according to the AddressBook class but in this test, I'm not testing whether data is valid, i.e. not testing the situation where <phone>abcdefg</phone>